samples/python/whisper_speech_recognition/whisper_speech_recognition.py

```diff
@@ -26,15 +26,19 @@ def streamer(word: str) -> bool:
         print(word, end="")
         return False
 
-    pipe.generate(
+    result = pipe.generate(
```

Comment:
can timestemp info somehow be streamed ? E.g. via custom WhispterStreamer which can have `put(timestemp)` in addition to `put(text)`

Reply:
Yes, sure. I'll think about public api and propose some variants. Can I address it in another PR? TODO added

Reply:
sure, thank you